packages/mobile/src/transactions/reducer.ts

```diff
@@ -61,6 +46,24 @@ type TransferTransactionTypes =
   | TransactionTypes.INVITE_RECEIVED
   | TransactionTypes.NETWORK_FEE
 
+export const isTransferType = (txType: TransactionTypes) => {
```

Comment:
Lmk if there's a cleaner way to check multiple types

Reply:
sometimes i will make an array here and then just check if the string is contained in the array

Reply:
Kk I find the function cleaner than the array so will keep as is

Reply:
Wondering if we can remove the duplication with TransferTransactionTypes here:
https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/transactions/reducer.ts#L52